extensions/binding/org.eclipse.smarthome.binding.tradfri/ESH-INF/i18n/tradfri_de.properties

```diff
@@ -0,0 +1,29 @@
+# binding
+binding.tradfri.name = TR�DFRI Erweiterung
+binding.tradfri.description = Diese Erweiterung integriert das IKEA TR�DFRI System. Durch diese k�nnen die TR�DFRI Lampen und Leuchten gesteuert werden. 
+
+# thing types
+thing-type.tradfri.gateway.label = TR�DFRI Gateway
+thing-type.tradfri.gateway.description = Steht f�r ein IKEA TR�DFRI Gateway.
+thing-type.tradfri.0100.label = Dimmbare Lampe
+thing-type.tradfri.0100.description = Steht f�r eine Lampe zum Dimmen.
+thing-type.tradfri.0220.label = Farbige Lampe
```

Comment:
Since the 0220 device is defined as color temperature light this should be something like `tuneable white`: `Farbtemperatur Lampe (weiß)`? wdyt?